src/js/jsx/sections/layers/LayerFace.jsx

```diff
@@ -554,8 +554,9 @@ define(function (require, exports, module) {
          * @private
          * @type {Droppable~onDragTargetMove}
          */
-        _handleDragTargetMove: function (draggedLayers, dragPosition) {
-            var isDropTarget = !draggedLayers.includes(this.props.layer),
+        _handleDragTargetMove: function (draggedLayerIds, dragPosition) {
+            var draggedLayers = this.props.document.layers.byIDs(draggedLayerIds),
```

Comment:
Instead of adding a new layerStructure method, can you use
```
var layerTree = this.props.document.layers,
     draggedLayers = draggedLayerIds.map(layerTree.byID, layerTree),
    ....
```